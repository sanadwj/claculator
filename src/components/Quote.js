import React from 'react';

const Quote = () => (
  <h2 className="quote">
    {/* eslint-disable-next-line max-len */}
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
  </h2>
);

export default Quote;
