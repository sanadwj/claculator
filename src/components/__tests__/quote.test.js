import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';

import Quote from '../Quote';

describe('Quote', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Router><Quote /></Router>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
