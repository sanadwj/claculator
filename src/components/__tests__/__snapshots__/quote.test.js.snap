// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote renders correctly 1`] = `
<h2
  className="quote"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</h2>
`;
